pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -231,12 +246,42 @@ def fractional_octave_bands(
         >>> ax.set_title(
         ...     "Filter bands and the sum of their squared magnitudes")
 
+    Show that the filters meet class I tolerance according to IEC 61260-1
+
+    .. plot::
```

Comment:
This plot does not appear on the current built at https://pyfar--829.org.readthedocs.build/en/829/modules/pyfar.dsp.filter.html#pyfar.dsp.filter.fractional_octave_bands. Do you know the reason?